src/react-components/ui-root.js

```diff
@@ -1912,93 +1922,91 @@ class UIRoot extends Component {
                 <FormattedMessage id="entry.leave-room" />
               </button>
             )}
-            {!this.state.frozen &&
-              !watching &&
-              !preload && (
-                <div
-                  className={classNames({
-                    [inviteStyles.inviteContainer]: true,
-                    [inviteStyles.inviteContainerBelowHud]: entered,
-                    [inviteStyles.inviteContainerInverted]: this.state.showShareDialog
-                  })}
-                >
-                  {!embed &&
-                    !streaming && (
-                      <button
-                        className={classNames({
-                          [inviteStyles.inviteButton]: true,
-                          [inviteStyles.hideSmallScreens]: this.occupantCount() > 1 && entered,
-                          [inviteStyles.inviteButtonLowered]: hasTopTip
-                        })}
-                        onClick={() => this.toggleShareDialog()}
-                      >
-                        <FormattedMessage id="entry.share-button" />
-                      </button>
-                    )}
-                  {showChooseSceneButton && (
+            {showInviteButton && (
```

Comment:
The diffs below are formatting changes as a result of using the `showFoo` constants defined above.